src/store/dashboard/ociDashboard/ociDashboardCommon.ts

```diff
@@ -14,23 +14,20 @@ export const ociDashboardTabFilters: OciFilters = {
 
 // eslint-disable-next-line no-shadow
 export const enum OciDashboardTab {
-  service_names = 'service_names',
+  product_services = 'product_services',
   payer_tenant_ids = 'payer_tenant_ids',
-  resource_locations = 'resource_locations',
+  regions = 'regions',
 }
 
 export interface OciDashboardWidget extends DashboardWidget<OciDashboardTab> {}
 
 export function getGroupByForTab(widget: OciDashboardWidget): OciQuery['group_by'] {
   switch (widget.currentTab) {
-    case OciDashboardTab.service_names:
-      // Use group_by for service tab and filter for others -- https://github.com/project-koku/koku-ui/issues/846
-      return {
-        product_service: widget.tabsFilter && widget.tabsFilter.service_name ? widget.tabsFilter.service_name : '*',
-      };
```

Comment:
On second thought, we should probably restore these lines. We want to apply the filter(s) defined by `widget.tabsFilter.product_service`.

```
case OciDashboardTab.product_services:
  return {
    product_service: widget.tabsFilter && widget.tabsFilter.product_service ? widget.tabsFilter.product_service : '*',
  };
```